doc/source/conf.py

```diff
@@ -167,50 +161,75 @@
 # the title page.
 #latex_logo = None
 
-# For "manual" documents, if this is true, then toplevel headings are parts,
-# not chapters.
-#latex_use_parts = False
+# Documents to append as an appendix to all manuals.
+#latex_appendices = []
+
+# If false, no module index is generated.
+latex_use_modindex = False
```

Comment:
iiuc, this option was renamed to latex_domain_indices

Reply:
yes `latex_use_modindex` was removed at 1.6. Sorry for oversight I will fix later.